packages/core/src/components/BreadCrumb/BreadCrumb.styles.tsx

```diff
@@ -1,6 +1,6 @@
 import styled from "@emotion/styled";
 import { theme } from "@hitachivantara/uikit-styles";
-import { HvTypography } from "index";
+import { HvTypography } from "components/Typography";
```

Comment:
Why not `import { HvTypography } from "components";`?

Reply:
Updated to import from `components`